### ISSUE TITLE
can't get villager milk

### ISSUE THREAD

**Issue by MetriCfan** [TangoTek/TekTopia-Community#427]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

** Server/Location **
What server were you playing on? Main(Survival) or Dev (Creative)?
What are the world coordinates of your town hall marker?

**Additional context**

**Comment by MetriCfan** [comment]
Started out by giving the ranchers player buckets. Once the blacksmith started providing buckets I am still getting non villager milk. I removed all player milk buckets and reset the ranchers to a different profession before returning them as ranchers. They only had access to village buckets and village cows but I still only get non-villager milk buckets. Have never had villager milk bucket.

**Comment by Hedkracka** [comment]
I have been getting player milk for a while now, even tho my smith has made buckets, and i have villager cows.

**Comment by spankymcmillan** [comment]
Ok this problem isn't just not getting villager milk.  My ranchers are literally producing the bucket for milk in a bucket (non-vill) out of thin air.  I checked all their inventories and storage room.  Each villager has 2 vill. buckets.  Store room has 3 vill. buckets.  No non-vill buckets.  They go to milk cows with 2 villager buckets.  Cow gets milked non-vill. milk in a bucket produced; however, the rancher still has the 2 vill. buckets he just had but now a 3rd bucket with milk has appeared.  I've got tons of buckets!

I put in a separate ticket for the duping.
